src/evaluation.tex

```diff
@@ -0,0 +1,5 @@
+\chapter{Evaluation}\label{sec:evaluation}
+After presenting your idea, you now have to devise a way of convincing the reader why your idea is great.
+You can do this by proofing the correctness of your algorithm, proving the soundness of your type system, a proof-of-concept compiler or a benchmark of your implementation compared against pre-existing solutions.
```

Comment:
Typo :)
```suggestion
You can do this by proving the correctness of your algorithm, proving the soundness of your type system, a proof-of-concept compiler or a benchmark of your implementation compared against pre-existing solutions.
```

Reply:
Hm language tool appears to have let me down in finding these errors :(